ducktape/cluster/json.py

```diff
@@ -23,11 +23,14 @@ class JsonCluster(Cluster):
     An implementation of Cluster that uses static settings specified in a cluster file.
     """
 
-    def __init__(self, cluster_json=None):
+    def __init__(self, cluster_json=None, *args, **kwargs):
         super(JsonCluster, self).__init__()
         if cluster_json is None:
-            cluster_json_path = os.path.abspath(os.path.join(os.getcwd(), "cluster.json"))
-            cluster_json = json.load(open(cluster_json_path))
+            # This is a directly instantiation of JsonCluster rather than from a subclass
+            cluster_file = kwargs.get("cluster_file")
```

Comment:
minor - this looks equivalent to
`cluster_file = kwargs.get("cluster_file", ConsoleConfig.CLUSTER_FILE)`